### DISCUSSION TITLE
Can we get Total Duration in Allure report Class Level

### DISCUSSION THREAD

**Discussion by cb-anomitromunshi** [allure-framework/allure2#2635] [Questions & Support] [answered]
we already get total duration in report Test methods wise. can we also get for class level?
If possibel then how?
<img width="468" alt="image" src="https://github.com/allure-framework/allure2/assets/94455471/06368235-6888-4f1f-ba12-a7839f1d95fa">

so basically for above it should be (11s 224 ms +14s 043ms)

**Comment by baev** [comment, marked as answer]
You can toggle the group time info by clicking on the `i` icon:

<img width="174" alt="Screenshot 2024-07-10 at 12 24 52" src="https://github.com/allure-framework/allure2/assets/2149631/8c21e498-5d95-4d2b-9ac7-c417b92e1878">

**Reply by cb-anomitromunshi** [reply]
Thank you :)
